sql/item_latents.sql

```diff
@@ -2777,22 +2776,298 @@ INSERT INTO `item_latents` VALUES(14646, 29, 10, 32, 0); -- Darksday: MDB+10
 -- -------------------------------------------------------
 -- LAVALIER
 -- -------------------------------------------------------
-INSERT INTO `item_latents` VALUES (10961,60,-40,13,14); -- CHARMED:LIGHTRES
-INSERT INTO `item_latents` VALUES (10961,61,-40,13,14); -- CHARMED:DARKRES
+INSERT INTO `item_latents` VALUES(10961,60,-40,13,14); -- CHARMED:LIGHTRES
```

Comment:
Please don't remove the space. these liens were otherwise unchanged, no reason to take that out. Its in the formatting mysql natively dumps to.